site/content/docs/5.0/migration.md

```diff
@@ -7,11 +7,26 @@ aliases: "/migration/"
 toc: true
 ---
 
+## v5.0.0-beta3
+
+### JavaScript
+
+- All plugins can now accept a CSS selector as the first argument. You can either pass a DOM element or any valid CSS selector to create a new instance of the plugin:
+
+  ```js
+  var modal = new bootstrap.Modal('#myModal')
```

Comment:
I guess that's a leftover of the merge, where you should use boosted instead of bootstrap ? :)